src/components/ExpensiTextInput/propTypes.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Input label */
+    label: PropTypes.string,
+
+    /** Input value */
+    value: PropTypes.string.isRequired,
+
+    /** Input value placeholder */
+    placeholder: PropTypes.string,
+
+    /** Input with error  */
+    error: PropTypes.bool,
```

Comment:
Usually it's clearer to prefix a boolean with `isXXX` or `hasXXX` or something to make it obvious it's a boolean.
```suggestion
    hasError: PropTypes.bool,
```